Makefile

```diff
@@ -37,21 +37,18 @@ BUILD_JOBS := $(addprefix build-,${ENVS})
 PUBLISH_JOBS := $(addprefix publish-,${ENVS})
 
 .PHONY: all
-all: build ## Prepare and run the project tests
+all: test build ## Prepare and run the project tests
 
 .PHONY: install
-install: publish-local ## Install the plugin binaries to the local machine
+install: test## Install the plugin binaries to the local machine
 	@# TODO avoid deleting an existing plugin once in place reinstalls are working again
 	@tanzu plugin delete apps > /dev/null 2>&1 || true
-	tanzu plugin install apps --version $(BUILD_VERSION) --local $(TANZU_PLUGIN_PUBLISH_PATH)/${GOHOSTOS}-${GOHOSTARCH}
-
-.PHONY: build-local
-build-local: prepare test ## Build the plugin binaries locally for the host architecture
-	echo BUILD_VERSION: $(BUILD_VERSION)
 	tanzu builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path ./cmd/plugin --target local --artifacts ${ARTIFACTS_DIR}/${GOHOSTOS}/${GOHOSTARCH}/cli
```

Comment:
is it possible here to call `build-${GOHOSTOS}-${GOHOSTARCH}`? instead writing the command again?

Reply:
builder command in local install has --target set as local , while the build -% PHONY has the target set as --target ${OS}_${ARCH}

Reply:
`local` can be interpreted as a shortcut for `runtime.GOOS, runtime.GOARCH` but I am good with leaving it a command to keep consistency in the `make` target